### ISSUE TITLE
[Reveal] Feature Request: Make appendTo("body") configurable

### ISSUE THREAD

**Issue by phifa** [zurb/foundation-sites#9046]
For ajax loading reasons I need all of me content inside a specific div. Problem is, the reveal plugins always gets attached to the body. When I change the code in line 88 in founation.reveal.js like this it works like a charm: 

```
  _makeOverlay(id) {
    var $overlay = $('<div></div>')
                    .addClass('reveal-overlay')
                    .appendTo('#configurableDiv');
    return $overlay;
  }
```

**Comment by Owlbertz** [comment]
That would be a nice enhancement for v6.3. You think you could create a PR for this?

**Comment by phifa** [comment]
like so @Owlbertz  https://github.com/zurb/foundation-sites/pull/9050 ?

**Comment by phifa** [comment]
I did it again, this time with the newest version -.- 

https://github.com/zurb/foundation-sites/pull/9094 @Owlbertz

**Comment by kball** [comment]
Fixed with #9094
